src/components/message/message-content/message-content.style.ts

```diff
@@ -1,7 +1,8 @@
 import styled from "styled-components";
 
-const MessageContentStyle = styled.div`
-  padding: 15px 50px 15px 20px;
+const MessageContentStyle = styled.div<{ showCloseIcon?: boolean }>`
+  padding: 15px ${({ showCloseIcon }) => (showCloseIcon ? "50px" : "20px")} 15px
```

Comment:
Ideally we should use any specified design tokens from DS over hardcoded values, however I couldn't seem to find any relevant tokens in the design from DS, so happy to approve. 

We can always go back and add the tokens in at a later point once we can determine which tokens to use and where to use them :)